src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/instrument-lambda-function/enable-serverless-monitoring-aws-lambda-layerless.mdx

```diff
@@ -148,15 +148,16 @@ If you're using a Lambda function without a layer, you'll need to instrument New
     id="node"
     title="Node.js"
   >
-    To instrument your Node.js Lambda:
+   To instrument your Node.js Lambda:
 
     1. Download our Node.js agent package and place it in the same directory as your function, ensuring the agent is installed as a dependency in the `node_modules` directory. Use the Node Package Manager:
 
        ```bash
        npm install newrelic --save
        ```
 
-    2. In your Lambda code, require the agent module at the top of the file, and wrap the handler function. For example:
+    2. In your Lambda code, require the agent module at the top of the file, and wrap the handler function with the newrelic `setLambdaHandler`. For example:
+       - Sample Code for CommonJS:
```

Comment:
Suggestion: use "*" instead of "-" for bulleted list.
 example:  * Sample Code for CommonJS: